runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -97,4 +97,14 @@
 	 * Send a private message.
 	 */
 	public static final int PRIVMSG = 10004;
+
+	/**
+	 * Spawns a fake experience drop
```

Comment:
This isn't entirely accurate. It's more accurately a "disabled" XP drop, for players who, if not for some circumstance, would have received some experience but didn't. I'm inclined to say that it would be valuable to include more information and context around this so it's clear what it's used for natively.